### PR TITLE
perf: featured drops

### LINKED ISSUE #9349: Faster loading for drop carousel
- it seems like the loading takes some time - wao mentioned something that the request is taking too long 

is there anything we can do to have it load quickly as it is the first thing the user sees

image

### LINKED ISSUE #9369: Speed up loading of drop cards on landing
Sometimes it's there like 5s and I'm on 1Gbit optics 😅 

If you can come up with some "caching" idea, it seems like those "edge" workers are always waiting for something ?
cc @preschian @vikiival 

image

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #9369
- [x] Closes #9349

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)

### CODE REVIEW

**Comment by preschian** [comment]
I removed some of the calls to our graphql queries. As long the data from our db is enough, it should be fine, I guess?

**Review by roiLeo — APPROVED** [review]
Noice! more legibility

would be cool to move `useDrops` inside composables folder, weird that we have to use `onMounted` to call fetch

File: components/drops/useDrops.ts [hunk 1]
```diff
@@ -55,41 +56,28 @@ export function useDrops(query?: GetDropsQuery) {
...
-          })
-        }),
-    ).then((dropsDataList) => {
-      drops.value = sortBy(dropsDataList as Drop[], (drop) =>
-        DROP_LIST_ORDER.indexOf(drop.status),
-      )
-      loaded.value = true
-    })
+    dropsList.value
+      .filter((drop) => !isProduction || drop.chain !== 'ahk')
```
Comment (Jarsen136) [inline components/drops/useDrops.ts:63]: Is it still necessary to have this filter as you have changed the config.

image

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 7.66 USD/DOT ~ 3.916 $DOT
🧗 [1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p](https://polkadot.subscan.io/account/1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)
🔗 [0x92d3a6e3a5ee03c002be039b2d06e9575843db8b90828f87c4e272d5cc0dd22a](https://polkadot.subscan.io/extrinsic/0x92d3a6e3a5ee03c002be039b2d06e9575843db8b90828f87c4e272d5cc0dd22a)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
> lmk if we want it more

I would easily add 1hr should be okay for most of cases, we are adding drop at least day before

**Pull request merged**